misc/docker_tests/base_images/fedora_20/Dockerfile

```diff
@@ -2,8 +2,11 @@ FROM fedora:20
 
 MAINTAINER Lion Krischer
 
-RUN rpm -Uvh http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-2.noarch.rpm
+RUN rpm -Uvh http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
```

Comment:
So, this is weird. I don't know why you'd enable EPEL repos on Fedora. I don't think anyone would normally do so. Usually, they'd be out of date.